AlfrescoApp/View Controllers/Search View Controller/SearchManager.h

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile iOS App.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+#import <Foundation/Foundation.h>
+#import "SearchResultsTableViewController.h"
+#import "SitesTableListViewController.h"
+
+@interface SearchManager : NSObject
```

Comment:
I have a concern that the class suffix "Manager" is used exclusively for singleton classes in the rest of the app codebase and this breaks the pattern. Perhaps "SearchService" is a better name that would remove the confusion?